src/Textarea/Textarea.test.tsx

```diff
@@ -81,4 +82,22 @@ describe('Textarea Component', () => {
     const { container } = render(<Textarea label="howdy" value="hi" />)
     expect(container.querySelector('.textarea')?.classList).toContain('has-value')
   })
+
+  it('should render the character limit correctly', () => {
+    render(<Textarea placeholder="Please enter limited characters here" characterLimit={20} />)
+
+    const textarea = screen.getByPlaceholderText('Please enter limited characters here')
+    expect(textarea).toBeInTheDocument()
+
+    // Add text within limit
+    userEvent.type(textarea, 'Character count')
+    expect(screen.getByText('5')).toBeInTheDocument()
+    expect(textarea).toHaveTextContent('Character count')
+
+    // Test the character limit
+    userEvent.type(textarea, ' limit')
+    expect(screen.getByText('0')).toBeInTheDocument()
+    expect(textarea).toHaveTextContent('Character count limi')
```

Comment:
I think limit is misspelled.

Reply:
Actually that is intended. The character limit is at 20 characters, so the full text of "Character count limit" will get cut off right at the last 't' character. So the it **should** say "Character count limi" as the final text result of the textarea.

Reply:
Ohhhh hahaah nice